source/Player.hx

```diff
@@ -16,24 +17,35 @@ class Player extends FlxSprite
 	public var xAccel:Float = 400;
 	public var xMaxSpeed(default, set):Float;
 	
+	/*	when pressing jump, the player will have thrust at the jumpSpeed
+		for a maximum of jumpThrustTime*/ 
+	public var jumpSpeed:Float = -300; //velocity at which player jumps.
+	public var jumpThrustTime:Float = 0.35; //time thrust is applied to player
+	public var jumpReleased:Bool = true; //button was released while on the ground
+	public var stoppedJumping = true; // if the a button was released while jumping.
+	public var jumpStartedTime:FlxTimer; //timer to limit jump time
+	
 	public var walkSpeed:Float = 100;
-	public var runSpeed:Float = 200;
+	public var runSpeed:Float = 250;
```

Comment:
I also have more general feedback on your changes, which I'll comment on after this little review.